modules/library/src/main/scala/zio/elasticsearch/RawItem.scala

```diff
@@ -0,0 +1,10 @@
+package zio.elasticsearch
+
+import zio.json.ast.Json
+import zio.schema.Schema
+import zio.schema.codec.DecodeError
+import zio.schema.codec.JsonCodec.JsonDecoder
+
+final case class RawItem(raw: Json) {
```

Comment:
Let's use `Document` here. Consider renaming the existing `Document` to something else.